### PR TITLE
Turkish translation

### PR DESCRIPTION
**Pull request by Saydemr**
Translated to Turkish. I am going to add the translation of the installation guide later.

### CODE REVIEW

**Comment by Saydemr** [comment]
I feel some ambiguity in the installation part of the translation because of lacking terminology in Turkish. I definitely need help from some experienced ones who are familiar with the terminology to have clearer flow.

Türkçede bazı terimlerin olmaması nedeniyle çeviride eksiklik hissediyorum. Özellikle kurulum kısmının deneyimli biri tarafından okunup, anlaşılmayan yerlerin -varsa eğer- düzenlenmesi gerekiyor.

**Comment by Saydemr** [comment]
My job ends here I guess. I tried translating [specific_topics.md](https://github.com/bitemyapp/learnhaskell/blob/master/specific_topics.md) a couple of times but I couldn't because I am also a beginner to Haskell. I believe other parts are well and ready to pull. @bitemyapp

File: guide-tr.md [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+
+İnternet söyleşi kanalımızın ağ istemcisi [için](http://webchat.freenode.net/).
+
+Haskell [e-posta listesi](https://wiki.haskell.org/Mailing_lists).
+
+### Topluluk Kuralları
+
+[Chris Done'ın eğitim hakkındaki gönderisine göz atın](http://chrisdone.com/posts/teaching)
+
+Nazik ve kibar olun. Kabalık ve edepsizlik insanları korkutur ve onları iletişime katkı yapmalarını engeller.
```
Comment (ozg-y) [inline guide-tr.md:51]: onların* bi de bence yapmalarını değil sağlamalarını

File: guide-tr.md [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+
+Yapıcı olmayan eleştiriler sadece söyleyene yarar sağlar, dinleyene değil.
+
+Olayları "çok basit" veya "önemsiz" olarak nitelendirmeyin. Bu şekilde insanlar sıkı çalışmaları gerektiği için kötü hissederler. Yavaş öğrenen kişiler genellikle titizlerdir ve bu övülmesi gereken bir şey!
```
Comment (ozg-y) [inline guide-tr.md:55]: şeydir

File: guide-tr.md [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+
+Şaşırmış gibi yapmak yok. Birisi size bir şeyi bilmediğini söyleyince şaşırmış numarası yapmayın. Bu durumda karşınızdaki berbat hisseder ve siz de kendinizi uyanık hissetmekten başka hiç bir şey elde etmezsiniz.
```
Comment (ozg-y) [inline guide-tr.md:57]: hiçbir şey*

**Pull request merged**